scripts/generate_code.py

```diff
@@ -561,5 +560,50 @@ def flatc_annotate(schema, include=None, data=None, cwd=tests_path):
     schema="monster_test.fbs", include="include_test", data="monsterdata_test.mon"
 )
 
-# Run the generate_grpc_examples script
-generate_grpc_examples.GenerateGRPCExamples()
```

Comment:
Please leave this as is, and undelete the two files.

Reply:
The code in two deleted file are redundant.

Reply:
Yes, its part of my plan to migrate stuff over, so it is duplicated on purpose.

Reply:
Ok, I will undelete those

Reply:
They are still deleted in the latest version

Reply:
They are now undeleted